bin/tictactoe

```diff
@@ -1,3 +1,33 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+
+
+def play
+	g = Game.new
+	g.start
+	
+	while !g.completed?
+		nil
+	end
+	
+	play_again?
+end
+
+#had to move play_again logic here because in game.rb it gave rspec errors because it asked for input
+def play_again?
+	puts "Would you like to play again? y/n"
+	input = gets.strip
+	case input
+	when 'y'
+		play
+	when 'n'
+		exit
+	else
+		puts "\nPlease enter 'y' or 'n'"
+		play_again?
+	end
+end
+
+play
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.